meal-planning/src/components/shopping-list/ShoppingList.js

```diff
@@ -7,68 +7,57 @@ import Checkbox from '@material-ui/core/Checkbox';
 import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
-import styles from "./ShoppingList.module.scss";
+import styles from './ShoppingList.module.scss';
+import RecipeService from '../../services/RecipeService';
 
 const ShoppingList = () => {
-  const [checked, setChecked] = React.useState([-1]);
-  const ingredients = ['2 bananas', '1/2 cup flour', '1 tsp salt', '1 tsp vanilla', '3 eggs'];
+  const ingredients = RecipeService.getIngredients();
   const [currentList, setList] = React.useState(ingredients);
```

Comment:
I think you already mentioned this, but this could be simplified into 1 line (getIngredients() as the parameter for useState). Honestly not sure if that's a best practice, but I don't see why not.